src/android/PurchasesPlugin.java

```diff
@@ -35,7 +35,7 @@ public class PurchasesPlugin extends AnnotatedCordovaPlugin {
     public static final String PLATFORM_NAME = "cordova";
     public static final String PLUGIN_VERSION = "3.0.0-rc.8";
 
-    @PluginAction(thread = ExecutionThread.UI, actionName = "configure", isAutofinish = false)
+    @PluginAction(thread = ExecutionThread.MAIN, actionName = "configure", isAutofinish = false)
```

Comment:
awesome! but let's definitely leave a comment here explaining why this is important